src/openforms/forms/api/drf_camel_case.py

```diff
@@ -17,7 +17,13 @@ def json_underscoreize(self):
            while we'd rather be able to just validate this depending on the applicable
            registration backend. Future improvement!
         """
-        ignore_fields = []
+        ignore_fields = [
+            # ServiceFetchConfiguration attributes
+            "body",
+            "headers",
+            "mapping_expression",
+            "query_params",
+        ]
```

Comment:
I'm not a fan of this global approach for the form parser due to the risk of key collision in the future - I think it'd be better to define a separate parser/renderer for the form variables related endpoints, similarly to `IgnoreConfigurationFieldCamelCaseJSONParser` and then specify that parser class in the `parser_classes` argument for the `action` [decorator](https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing).